### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tiagopog**
Just fixed some typos it the current doc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -739,7 +739,7 @@ end.get
 
 ## Users
 
-Many apps have a unified login that works across the mobile app and other systems. Accessing user accounts through parse-ruby-client lets you build this functionality on top of Parse.
+Many apps have an unified login that works across the mobile app and other systems. Accessing user accounts through parse-ruby-client lets you build this functionality on top of Parse.
```
Comment (Xavdidtheshadow) [inline README.md]: "a" is actually correct here

File: README.md [hunk 2]
```diff
@@ -1197,7 +1197,7 @@ end.save
 
 TODO: Implement this!
 
-Now that you have a bunch of objects with spatial coordinates, it would be nice to find out which objects are closest to a point. This can be done by using a GeoPoint data type with query on the field using $nearSphere. Getting a list of ten places that are closest to a user may look something like:
+Now that you have a bunch of objects with spatial coordinates, it would be nice to find out which objects are closest to a point. This can be done by using a GeoPoint data type with query on the field using $nearSphere. Getting a list of ten places that are closest to an user may look something like:
```
Comment (Xavdidtheshadow) [inline README.md]: "a" is also correct here

File: README.md (hunk 1 above)
Comment (tiagopog) [inline README.md]: [Indeed](http://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user)

File: README.md (hunk 2 above)
Comment (tiagopog) [inline README.md]: [Indeed](http://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user)

File: README.md [hunk 3]
```diff
@@ -711,7 +711,7 @@ end.get
 
 ### Counting Objects
 
-If you are limiting your query, or if there are a very large number of results, and you want to know how many total results there are without returning them all, you can use the `count` parameter. For example, if you only care about the number of games played by a particular player:
```
Comment (Xavdidtheshadow) [inline README.md]: per [this](https://english.stackexchange.com/questions/51729/is-the-sentence-there-is-a-large-number-of-labourers-who-want-to-migrate-to-jap) I believe "are" is correct as well.

Comment (tiagopog) [inline README.md]: Good explanation, thanks.

**Pull request merged**